api/app/Http/Resources/UserResource.php

```diff
@@ -65,6 +65,8 @@ public function toArray($request)
             'hasDisability' => $this->has_disability,
             'isIndigenous' => $this->is_indigenous,
             'isVisibleMinority' => $this->is_visible_minority,
+            'indigenousCommunities' => $this->indigenous_communities,
```

Comment:
I did this and reverted it in https://github.com/GCTC-NTGC/gc-digital-talent/pull/4994/commits/7445fe6155a889e809c203dd34f3870e6bb375a5
because I think it is a not-so-good idea

unit test fails because the snapshot created does not match the expected snapshot, the expected one comes from this file

![image](https://user-images.githubusercontent.com/40485260/205733953-0afcb5b0-e4ae-45da-958c-b31ae28aa105.png)

I can change `PoolCandidateSnapshot` by removing the field from it too, but then it no longer is in sync with `userOperations` in the frontend

Reply:
Yay, the test worked!  Yeah, we need to keep them in sync.  I guess we can make this change with the frontend then.